airflow/utils/platform_utils.py

```diff
@@ -0,0 +1,16 @@
+import platform
+
+def is_windows() -> bool:
+    """
+    Returns true if executing system is Windows
+    """
+    return platform.system() == 'Windows'
```

Comment:
Would be a good idea to `functools.lru_cache` this.

Reply:
Obsolete now since we are using a constant in the latest commit but I will keep this in mind.